src/poll.ts

```diff
@@ -236,17 +250,36 @@ namespace Poll
 
 			return null;
 		}
+		
+		let userCount: number = 0;
+		for (let member of server.members.array())
+		{
+			if (member.user.presence.status === "online")
```

Comment:
The one issue with only checking online users is that if a user waits until the majority of the server is offline, they could easily kick people or do anything that requires a 2/3 majority of online users.

Reply:
I was thinking of adding a minimum limit, but then they could just wait until it's all people of their opinion. My only worry with all users is some people will never come on. Maybe 2/3 online, 6, or 2/3s total, whichever is higher.

Or maybe have a timeout on the vote, and do the action if a minimum number of votes have been placed?

Another issue with this is that people could come online right after the poll is started, and the number would be lower (I guess this is where those lambdas come in)

Reply:
For now, I think we can make it like this: require half of the server to be online, and require 2/3 of the online users to upvote. We can change this later, but let's just put something down for now.

We can deal with timeouts on votes later. That's another issue I've been meaning to address, as a poll could go on indefinitely if it isn't canceled or passed.

This is why I made `Poll.VotesNeeded` a lambda.